Gems/AtomLyIntegration/EMotionFXAtom/Code/Source/AtomActorDebugDraw.cpp

```diff
@@ -266,7 +280,9 @@ namespace AZ::Render
                 debugDisplay,
                 instance->GetActor()->GetPhysicsSetup()->GetColliderConfigByType(EMotionFX::PhysicsSetup::SimulatedObjectCollider),
                 nodeDebugDrawDataFunction,
-                { renderActorSettings.m_simulatedObjectColliderColor, renderActorSettings.m_selectedSimulatedObjectColliderColor });
+                { renderActorSettings.m_simulatedObjectColliderColor,
```

Comment:
suggestion: This is minor and not 100% directly related to your change, but can you add the type of the initializer here for the braced initializer - it makes tracking down type usages so much easier in future (trust me, I know 😅)

Reply:
Good idea, done!